test/controllers/return-submissions.controller.test.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, before, beforeEach, afterEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Things we need to stub
+const ViewReturnSubmissionService = require('../../app/services/return-submissions/view-return-submission.service.js')
+
+// For running our service
+const { init } = require('../../app/server.js')
+
+describe('Return Submissions controller', () => {
+  let options
+  let server
+
+  // Create server before running the tests
+  before(async () => {
+    server = await init()
+  })
+
+  beforeEach(async () => {
```

Comment:
Doesn't need to be async
```suggestion
  beforeEach(() => {
```